docs/modules/secret-operator/pages/secretclass.adoc

```diff
@@ -219,6 +219,44 @@ If the same AD domain _is_ shared between multiple Kubernetes clusters, the foll
 * The Kubernetes Nodes' names and fully qualified domain names
 * The Kubernetes Namespaces' names (only Namespaces that use Kerberos)
 
+[#ad-acl]
+===== Access Control
+
+The Secret Operator needs permission to create users in the configured base container
+(`kerberosKeytab.admin.activeDirectory.userDistinguishedName`), as well as to reset their passwords.
+
+This can be configured using the following PowerShell script:
+
```

Comment:
It might make sense to also have a textual description for people who want to click this instead of running a PowerShell script.
Doesn't need to be guide "click here, then click there" - just "account needs these rights configured"

Reply:
Isn't that covered by the text above? Honestly not sure about how to flesh it out better without going full tutorial mode. :/

Reply:
The text sounded so prose-like, I was hoping there might be some technical names for the privileges ore something like that ..  I mean .. I guess there is, its apparently `00299570-246d-11d0-a768-00aa006e0529` :)

I presume in the gui/wizard/whereever you could click this its just called "change password" ?

Reply:
In the GUI it's all grouped in under "All extended rights".

Reply:
It's also.. very hidden away that you can set it at all (need to use the low-level ADSI Edit tool to configure ACLs at all).

Reply:
3f6896cc5b6c7c8b63bb237d3d7f546b1f8b2f99

Reply:
In "Active Directory Users and Computers" there are no permission options at all. "ADSI Edit" is a lower-level graphical tool that does expose them.. but is still missing a lot of extended rights (which, yes, just get grouped under "all extended rights").